synapse/lib/webapp.py

```diff
@@ -102,14 +102,20 @@ def getFooByBar(self, bar):
         wapp.main()
 
     '''
-    def __init__(self, **settings):
+    def __init__(self, app_config=None, srv_config=None):
+
+        if not app_config:
+            app_config = {}
+
+        if not srv_config:
+            srv_config = {}
+
         EventBus.__init__(self)
         s_daemon.DmonConf.__init__(self)
-        tornado.web.Application.__init__(self, **settings)
+        tornado.web.Application.__init__(self, **app_config)
 
         self.loop = tornado.ioloop.IOLoop()
-        self.serv = tornado.httpserver.HTTPServer(self)
-
+        self.serv = tornado.httpserver.HTTPServer(self, **srv_config)
         self.boss = s_async.Boss()
 
         # FIXME options
```

Comment:
I think we should maybe keep `**settings` on the end and use that to specify things like the boss pool size (hit expand to see the boss code below this line)